### ISSUE TITLE
File description and metadata lack sufficient color contrast

### ISSUE THREAD

**Issue by Hanal-code** [techmatters/terraso-web-client#393]
## Description
Shared file description and the metadata's text color doesn't meet the color contrast requirement. 
The alignment of the file description is off as well. 

## Steps To Reproduce
Steps to reproduce the behavior. Describe what you typed, clicked, tapped, swiped, saw and heard.
1. Go to URL: https://app.staging.terraso.org/groups/academic-research-test OR any group/landscape with a shared file
2. See the file's metadata in light gray. 

## Expected behavior
COLOR: #666666
<img width="341" alt="image" src="https://user-images.githubusercontent.com/92342777/171640928-bb9b5ba9-a8c8-4326-b241-9299203c3826.png">

## Actual behavior
<img width="425" alt="image" src="https://user-images.githubusercontent.com/92342777/171640705-92a1cadd-cf97-4910-a125-9e69be06d506.png">
